util/progress/progressui/printer.go

```diff
@@ -63,6 +64,9 @@ func (p *textMux) printVtx(t *trace, dgst digest.Digest) {
 		if p.notFirst {
 			fmt.Fprintln(p.w, "")
 		} else {
+			if p.desc != "" {
+				fmt.Fprintf(p.w, "#0 %s\n\n", p.desc)
```

Comment:
Is there we could right-align the p.desc text?

So instead of:

```
$ docker buildx build .
[+] Building 2.6s (9/14)                                        docker-container:builder2
 => [internal] load build definition from Dockerfile            0.1s
 => => transferring dockerfile: 2.85kB                          0.1s 
 => [internal] load .dockerignore                               0.1s 
 => => transferring context: 45B                                0.1s 
 => resolve image config for docker.io/docker/dockerfile:1.4    0.4s
```

we could have:

```
$ docker buildx build .
[+] Building 2.6s (9/14)                   docker-container:builder2
 => [internal] load build definition from Dockerfile            0.1s
 => => transferring dockerfile: 2.85kB                          0.1s 
 => [internal] load .dockerignore                               0.1s 
 => => transferring context: 45B                                0.1s 
 => resolve image config for docker.io/docker/dockerfile:1.4    0.4s
```

Not sure, but ideally we should have the timing counts at the very end of the line.

Reply:
I find it currently more readable imo. WDYT @tonistiigi?

Reply:
It wastes too much space without being aligned to the right.

In `progress=plain` I think this is more like a debug log.

Reply:
Ah yes in console mode indeed it's not that great after all.

Reply:
Now aligned to the right in console mode (see updated PR description).